src/driverApp/controllers.js

```diff
@@ -0,0 +1,52 @@
+import handleResponse from '../controllers/handleResponse.js';
+import User from '../models/user.js';
+import passwordGenerator from '../utils/generatePassword.js';
+import sendRegisterEmail from '../utils/sendRegisterEmail.js';
+import { Driver } from './models.js';
+
+export const createDriver = async (req, res) => {
+	const passwords = await passwordGenerator();
+	const newUser = User.create({
+		email: req.body.email,
+		password: passwords.hashedPassword,
+		firstName: req.body.firstName,
+		lastName: req.body.lastName,
+		role: 'driver',
+	});
+	const newDriver = Driver.create(req.body);
+	newDriver.user = newUser;
+	await newDriver.save();
+	await sendRegisterEmail(
+		'https://phantom-codebandits-staging.herokuapp.com/login',
+		newUser.lastName,
+		newUser.email,
+		passwords.plainPassword
+	);
+	return handleResponse(res, 201, newDriver);
+};
+
+export const getDrivers = async (req, res) => {
+	const drivers = await Driver.find();
+	if (drivers.length < 1) return handleResponse(res, 200, res.__('zero_users'));
+	return handleResponse(res, 200, drivers);
+};
```

Comment:
Add test for these lines or go back and recheck your controller's test if you tested in a perfect way.